api/core/v2/apikey.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "github.com/gogo/protobuf@v1.2.1/gogoproto/gogo.proto";
+import "meta.proto";
+
+package sensu.core.v2;
+
+option go_package = "v2";
+option (gogoproto.populate_all) = true;
+option (gogoproto.equal_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.testgen_all) = true;
+
+// An APIKey is an api key specification.
+message APIKey {
+  option (gogoproto.face) = true;
+  option (gogoproto.goproto_getters) = false;
+
+  // Metadata contains the name of the APIKey.
```

Comment:
I think we wanted to at least make use of the `Name` field, however not `Namespace` since these are cluster-wide resources. I can update the comment here 😄